Client/Components/ComparisonComponent.razor.cs

```diff
@@ -34,7 +34,14 @@ public partial class ComparisonComponent
             var countryData = country.Data;
             if (countryData != null)
             {
-                return countryData.Where(d => d.Name == ResourceType).First();
+                try
+                {
+                    return countryData.Where(d => d.Name == ResourceType).First();
```

Comment:
Should be first or default since we don't check for null and are fine with returning null values. The default in this case would be null